self-hosted/distributed-hypertables/about-distributed-hypertables.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: About distributed hypertables
-excerpt: Learn how distributed hypertables work in multi-node Timescale
+excerpt: Distributed hypertables are hypertables that span multiple nodes. Learn how distributed hypertables work in a self-hosted multi-node TimescaleDB instance
```

Comment:
For all distributed hypertables and multinode, can you change the excerpt to say" Sunsetted v2.14.x

Reply:
Done